.travis.yml

```diff
@@ -11,10 +11,12 @@ before_cache:
 stages:
   - style
   - test
-
+ - name: deploy
+  # require any tag name
+  if: tag =~ .*
 _before_install: &_before_install
-  - sudo apt-get update -y && apt install -y libsnappy-dev
-
+  - sudo apt-get update -y
+  - apt install -y libsnappy-dev
```

Comment:
If this works without sudo, then we need to remove it from everywhere else

Reply:
It didn't work without sudo the last time I tried